web/templates/shared/_form_errors.html.eex

```diff
@@ -1,8 +1,8 @@
 <%= if display_errors?(@form) do %>
   <div class="alert alert-danger">
   <p>Oops, something went wrong! Please check the errors below:</p> <ul>
-  <%= for {attr, message} <- @form.errors do %>
-    <li><%= humanize(attr) %> <%= message %></li>
-  <% end %> </ul>
+    <%= for {key, {value, _}} <- @form.errors do %>
```

Comment:
Instead of `[key: error]` the new format is `[key, {error, [translation_stuff]}]` which means that `error` could be a translation key and the `translation_stuff` list is options that would be passed into the translation.

Right now we're not using `gettext` due to the limited scope of the app so we're ignoring the extra.s